src/main/java/AlexLink/HomeWork/AbstractClass/CargoCar.java

```diff
@@ -0,0 +1,41 @@
+package AlexLink.HomeWork.AbstractClass;
+
+public class CargoCar extends GroundTransport {
+
+    private int loadCapacity;
+
+    public CargoCar(int power, int maxSpeed, int weight, String model, int countWheel, int fuelConsumption, int loadCapacity) {
+        super(power, maxSpeed, weight, model, countWheel, fuelConsumption);
+        if (loadCapacity > 0 && loadCapacity < 10000) this.loadCapacity = loadCapacity;
+    }
+
+    public int getLoadCapacity() {
+        return loadCapacity;
+    }
+
+    public void setLoadCapacity(int loadCapacity) {
+        if (loadCapacity > 0 && loadCapacity < 10000)
```

Comment:
Are you sure that you need all these conditions in setter-methods?

Reply:
fixed